rfc_frontmatter.md

```diff
@@ -156,4 +156,28 @@ This document defines FFV1, a lossless intra-frame video encoding format. FFV1 i
   </front>
 </reference>
 
+<reference anchor="FFV1_V0" target="https://git.videolan.org/?p=ffmpeg.git;a=commit;h=b548f2b91b701e1235608ac882ea6df915167c7e">
+  <front>
+    <title>Commit to mark FFV1 version 0 as non-experimental</title>
+    <author initials="M." surname="Niedermayer" fullname="Michael Niedermayer"/>
+    <date month="April" year="2006" />
+  </front>
+</reference>
+
+<reference anchor="FFV1_V1" target="https://git.videolan.org/?p=ffmpeg.git;a=commit;h=68f8d33becbd73b4d0aa277f472a6e8e72ea6849">
+  <front>
+    <title>Commit to release FFV1 version 1</title>
+    <author initials="M." surname="Niedermayer" fullname="Michael Niedermayer"/>
+    <date month="April" year="2009" />
+  </front>
+</reference>
+
+<reference anchor="FFV1_V3" target="https://git.libav.org/?p=libav.git;a=commit;h=fe6e5cbea7dbd5d2c67d79b5570e26debb70e95b">
+  <front>
+    <title>Commit to mark FFV1 version 3 as non-experimental</title>
+    <author initials="L." surname="Barbato" fullname="Luca Barbato"/>
+    <date month="June" year="2016" />
```

Comment:
This is incorrect, in fact all of the 1.3 entry
1.3 was made non experimental in 2013 by me
commit abe76b851c05eea8743f6c899cbe5f7409b0f301
Author: Michael Niedermayer <michaelni@gmx.at>
Date:   Sat Aug 17 04:28:53 2013 +0200

    ffv1enc: Make ffv1.3 non experimental